core/modules/locale/locale.pages.inc

```diff
@@ -320,9 +339,10 @@ function locale_translate_edit_form($form, &$form_state, $lid) {
   // list. The source language is English for built-in strings and the default
   // language for other strings.
   $languages = language_list();
-  if (!locale_translate_english()) {
```

Comment:
That removal was very likely unintentional.

Reply:
@indigoxela I removed it because I added back the code just below this, including `unset($languages[($omit)]);`

Reply:
`if (!locale_translate_english()` is a new feature added later to "translate English" - which means, override strings from code via locale.

I still belief, this shouldn't get removed, see this commit a7c9259979 for details.

Reply:
I don't know enough about this to say otherwise. If I revert this, some tests stop breaking. I'll revert this part.